SofaScoreAcademy/SofaScoreAcademy/Views/MatchView.swift

```diff
@@ -0,0 +1,241 @@
+            teamTwoScoreLabel.topAnchor.constraint(equalTo: teamOneScoreLabel.bottomAnchor, constant: 4),
+            teamTwoScoreLabel.widthAnchor.constraint(equalToConstant: 32.0),
+            teamTwoScoreLabel.heightAnchor.constraint(equalToConstant: 16.0),
+        ])
+        
+        NSLayoutConstraint.activate([
+            teamOneImageView.leadingAnchor.constraint(equalTo: eventCellsEventCellView.leadingAnchor, constant: 80),
+            teamOneImageView.topAnchor.constraint(equalTo: eventCellsEventCellView.topAnchor, constant: 10),
+            teamOneImageView.widthAnchor.constraint(equalToConstant: 16.0),
+            teamOneImageView.heightAnchor.constraint(equalToConstant: 16.0),
+        ])
+        
+        NSLayoutConstraint.activate([
+            teamTwoImageView.leadingAnchor.constraint(equalTo: eventCellsEventCellView.leadingAnchor, constant: 80),
+            teamTwoImageView.topAnchor.constraint(equalTo: teamOneScoreLabel.bottomAnchor, constant: 4),
+            teamTwoImageView.widthAnchor.constraint(equalToConstant: 16.0),
+            teamTwoImageView.heightAnchor.constraint(equalToConstant: 16.0),
+        ])
+        
+        NSLayoutConstraint.activate([
+            time.leadingAnchor.constraint(equalTo: eventCellsEventCellView.leadingAnchor, constant: 4),
+            time.topAnchor.constraint(equalTo: eventCellsEventCellView.topAnchor, constant: 10),
+            time.widthAnchor.constraint(equalToConstant: 56.0),
+            time.heightAnchor.constraint(equalToConstant: 16.0),
+        ])
+        
+        NSLayoutConstraint.activate([
+            status.leadingAnchor.constraint(equalTo: eventCellsEventCellView.leadingAnchor, constant: 4),
+            status.topAnchor.constraint(equalTo: teamOneScoreLabel.bottomAnchor, constant: 4),
+            status.widthAnchor.constraint(equalToConstant: 56.0),
+            status.heightAnchor.constraint(equalToConstant: 16.0),
+        ])
+        
+        NSLayoutConstraint.activate([
+            verticalDivider.leadingAnchor.constraint(equalTo: eventCellsEventCellView.leadingAnchor, constant: 63),
+            verticalDivider.topAnchor.constraint(equalTo: eventCellsEventCellView.topAnchor, constant: 8),
+            verticalDivider.widthAnchor.constraint(equalToConstant: 1.0),
+            verticalDivider.heightAnchor.constraint(equalToConstant: 40.0),
+        ])
+    }
+
+    func configure(with event: Event) {
+        teamOneLabel.text = event.teamOneName
+        teamTwoLabel.text = event.teamTwoName
+        time.text = event.formattedStartTime
+        
+        let teamOneScore = event.teamOneScore ?? 0
+        let teamTwoScore = event.teamTwoScore ?? 0
+            
+        teamOneScoreLabel.text = "\(teamOneScore)"
+        teamTwoScoreLabel.text = "\(teamTwoScore)"
+        
+        
+        switch event.matchStatus {
+            
+            case .inProgress:
```

Comment:
mislim da su tu prazni redovi overkill i negdje ti je indetation sjeban pa to popravi. takoder nemoras pisat UIFont i UIColor tu nigdje.